forgerock-auth/src/main/java/org/forgerock/android/auth/webauthn/WebAuthnRegistration.kt

```diff
@@ -159,13 +159,15 @@ open class WebAuthnRegistration() : WebAuthn() {
         //Extension to support username-less
         if (options.authenticatorSelection?.requireResidentKey == true &&
             options.authenticatorSelection?.residentKeyRequirement == ResidentKeyRequirement.RESIDENT_KEY_DISCOURAGED) {
-            val source = PublicKeyCredentialSource.builder()
-                .id(publicKeyCredential.rawId)
-                .rpid(options.rp.id)
-                .userHandle(Base64.decode(options.user.id, Base64.URL_SAFE or Base64.NO_WRAP))
-                .otherUI(options.user.displayName).build()
-            persist(context, source)
-        }
+            publicKeyCredential.rawId?.let {
```

Comment:
Is this an intentional change?

Reply:
+1

Reply:
Yes, it is intended, after upgrade to the new fido library, the getRawId  can return null.
![CleanShot 2024-09-17 at 12 35 56@2x](https://github.com/user-attachments/assets/9a0576f0-362d-4fb5-8d40-ca41fa0a7fca)